app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Teacher training bursaries"
+title: "Teacher training bursaries and scholarships"
```

Comment:
I think this is too long - by the time we have appended ` | Get Into Teaching GOV.UK`, the end of the title will be truncated in search results. It's more important to talk about bursaries as that is by far the bigger search term, plus we don't want to truncate GOV.UK as that offers reassurance

Reply:
Will change